README.md

```diff
@@ -53,7 +61,7 @@ You will be prompted to install the Python dependencies during the first use.
   - Temperature sensor
   - 7 Capacitive Touch sensors
 
-The simulator supports most of the sensors on CPX except **IR transmitter & Receiver**, **Sound Sensor (microphone)**, **Speaker (Play Tone)** and the **"tap" on Motion Sensor**.
+The simulator supports most of the sensors on CPX except **IR transmitter & Receiver**, **Sound Sensor (microphone)**, **Speaker (Play Tone)** and the **"tap" on Motion Sensor**. 

```

Comment:
_&ast;nit&ast;_
Remove whitespace at the end of the line.

Reply:
good catch